### DISCUSSION TITLE
Remove/Hide Billing Adress field on payment sheet

### DISCUSSION THREAD

**Discussion by AfaqaL** [flutter-stripe/flutter_stripe#1371] [Q&A]
Is there a way to use pre-build payment sheet but hide the billing address field (leave the card detail fields only). Or alternatively, to set up a custom card input fields an feed them to the stripe sdk?

**Comment by Caresle** [comment]
@AfaqaL for hide the billing address set the option `billingDetailsCollectionConfiguration` inside the `SetupPaymentSheetParameters` 
example:
```dart
await Stripe.instance.initPaymentSheet(
      paymentSheetParameters: SetupPaymentSheetParameters(
        customFlow: false,
        merchantDisplayName: 'Eccommerce Store',
        paymentIntentClientSecret: data.clientSecret,
        style: ThemeMode.light,
        billingDetailsCollectionConfiguration: BillingDetailsCollectionConfiguration(
          address: AddressCollectionMode.never
        )
      )
    );
```
Without the `billingDetailsCollectionConfiguration`
![image](https://github.com/flutter-stripe/flutter_stripe/assets/66541547/b46a6f20-51c0-4886-8e08-edd16d15b352)

With the `billingDetailsCollectionConfiguration` set
![image](https://github.com/flutter-stripe/flutter_stripe/assets/66541547/b6ec5ffd-bd77-4677-be31-331831c790f6)

**Comment by Anonymousgaurav** [comment]
can you please update the class in the verison 2.5.0. i didn't get this class 
 billingDetailsCollectionConfiguration: BillingDetailsCollectionConfiguration(
          address: AddressCollectionMode.never
        )
